Yoast/javascript
# Added missing React key attributes.

## Summary


This PR can be summarized in the following changelog entry:

* [schema-blocks] Fixes a bug where an error was shown in the browser's console when a job posting block was added to the post.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Create a new post.
* Add a job posting block.
* Check your browser's console, it should not have errors like these:
  ```
  Warning: Each child in a list should have a unique "key" prop.
  ```

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * N/A: behind a feature flag.

## UI changes

## Quality assurance


Fixes #
